tests/cpp/operator/mkldnn.cc

```diff
@@ -515,33 +557,34 @@ void PrintVerifyMsg(const NDArrayAttrs &arr1, const NDArrayAttrs &arr2) {
  *
  *  num_inputs / dim arguments used to scale shape (used for concat backwards to enlarge input shapes)
  */
-std::vector<NDArrayAttrs> GetTestInputArrays(bool rand = false, int num_inputs = 1, int dim = 0) {
+std::vector<NDArrayAttrs> GetTestInputArrays(bool rand = false,
+    std::vector<float> scale = {1}) {
```

Comment:
can you update the comments on `scale`? previously, you only scale on one dimension. why do you want to scale on multiple dimensions?